cmd/startnode.go

```diff
@@ -53,8 +53,10 @@ var StartnodeCmd = &cobra.Command{
 		}
 
 		args, md := node.FlagsToArgs(flags, sanitize.Path(datapath), false)
-		// Set flags to default for next `startnode` call
-		flags = node.DefaultFlags()
+		defer func() {
+			// Set flags to default for next `startnode` call
+			flags = node.DefaultFlags()
+		}()
```

Comment:
What's the purpose of this change?

Reply:
without it the node.DefaultFlags was overwriting the data in flags before using it.